src/docs/Components/DataTable/previews/sticky.preview.scss

```diff
@@ -0,0 +1,41 @@
+@import "settings-tools/_all-settings";
+
+@include import-font-families();
+
+@import "components/_c.badge";
+@import "components/_c.button";
+@import "components/_c.checkbox";
+@import "components/_c.text-input";
+@import "components/_c.left-icon-input";
+@import "components/_c.links";
+@import "components/_c.select";
+@import "components/_c.pagination";
+@import "components/_c.tag";
+@import "components/_c.toggle";
+@import "components/_c.datatable"; // mandatory
+@import "components/_c.datatable-tools"; // mandatory
+
+html,
+body {
+  overflow: auto !important;
+}
+
+.example {
+  padding: $mu125;
+}
+
+.mc-datatable {
+  $parent: &;
+
+  &--sticky {
```

Comment:
If the max-height is necessary, maybe you can add it to the doc?

Reply:
You are right. I just fixed it.